### ISSUE TITLE
Keybinds

### ISSUE THREAD

**Issue by cxw42** [cxw42/TabFern#28]
@r4j4h Following on from #8, which was merged in the course of the push to v0.1.
Besides the TODO item there, the other thing I would appreciate along with the code is a bit more documentation. At this point, the options and the default bindings don't look to me like they would tell a new user what the shortcuts were for.  Thanks!

**Comment by bluikko** [comment]
The confirmation dialog for closing a window should also have shortcut keys like &Yes, &No, &Cancel instead of Yes, No, Cancel.

**Comment by cxw42** [comment]
@bluikko Agreed.  For now, Enter and Escape work for Yes and Cancel, respectively, and you can turn off the dialog from the Options.  I also plan to add a checkbox for "Don't ask me again."

**Comment by cxw42** [comment]
Requests from @philiprhoades:

- CTRL-Home and CTRL-End to move the window to the top or the bottom (like TO)
- CTRL-Up/Down to move the window up or down a position in the tree (like TO)
- Home and End for just jumping to the top and bottom of a very long tree - like I have in TO at the moment but will gradually become an issue for me in TFS as I move stuff over . .

**Comment by SamHasler** [comment]
to expand on the above list, I think it would help with discovery if they shortcuts were listed on the right-click menu

1. <kbd>CTRL</kbd>+<kbd>Home</kbd> and move the window to the top (like TO)
   - Add shortcut description to right-click menu
2. <kbd>CTRL</kbd>+<kbd>End</kbd> to move the window to the bottom (like TO)
   - Add shortcut 
   - Add right click menu item 
   - Add shortcut description to right-click menu
4. <kbd>CTRL</kbd>+<kbd>Up</kbd>/<kbd>Down</kbd> to move the window up or down a position in the tree (like TO)
   - Add shortcut
   - Add right click menu item 
   - Add shortcut description to right-click menu
5. <kbd>Home</kbd> and <kbd>End</kbd> for just jumping to the top and bottom of a very long tree - like I have in TO at the moment but will gradually become an issue for me in TFS as I move stuff over . .
   - Add shortcut
   - Add right click menu item 
   - Add shortcut description to right-click menu
1. <kbd>F2</kbd> to edit an item (Window Name or Tab Note)
   - Add shortcut
   - Add shortcut description to right-click menu
1. <kbd>B</kbd> toggle border
   - Add shortcut
   - Add shortcut description to right-click menu
1. <kbd>Enter</kbd> to Open item
    I feel a right click menu item to document this would be needlessly cluttering the interface. Should we have a someone else to document keyboard shortcuts?

**Comment by cxw42** [comment]
@SamHasler thanks for the detailed proposal!
